uint32 id
float64[] pos
float64 duration
uint32 robots_needed
uint32[] predecessors
